pdf2image/pdf2image.py

```diff
@@ -174,12 +192,25 @@ def _parse_format(fmt):
     # Unable to parse the format so we'll use the default
     return 'ppm', parse_buffer_to_ppm, False
 
-def _page_count(pdf_path, userpw=None):
+
+def _get_command_path(command, poppler_path=None):
+    if platform.system() == 'Windows':
+        command = command + '.exe'
+
+    if poppler_path is not None:
+        command = poppler_path + command
+
+    return command
+
+
+def _page_count(pdf_path, userpw=None, poppler_path=None):
     try:
+        command = [_get_command_path("pdfinfo", poppler_path), pdf_path]
+
         if userpw is not None:
-            proc = Popen(["pdfinfo", pdf_path, '-upw', userpw], stdout=PIPE, stderr=PIPE)
-        else:
-            proc = Popen(["pdfinfo", pdf_path], stdout=PIPE, stderr=PIPE)
+            command = command + [userpw]
```

Comment:
This does not work, see https://travis-ci.org/Belval/pdf2image/jobs/509002906

You need to add the `-upw` parameter.